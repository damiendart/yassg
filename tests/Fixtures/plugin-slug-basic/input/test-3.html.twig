{#---
  slug: '/this-should-not-be-changed'
---#}
<h1>{{ slug }}</h1>
