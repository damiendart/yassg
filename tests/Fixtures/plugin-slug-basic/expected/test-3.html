<h1>/this-should-not-be-changed</h1>
